index.html

```diff
@@ -11,6 +11,68 @@
 </head>
 
 <body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
-</body>
+    <div class="navigation">
+        <a href="">Services</a>
+        <a href="">Product</a>
+        <a href="">Vision</a>
```

Comment:
I would suggest wrapping all your `<a>` tags for your nav bar inside a wrapper tag (a `<nav>` tag would work well here) so that you can manipulate them in a flexbox separate from your nav logo. Doing so will be very helpful for matching the design file once you're setting CSS properties inside media queries.